### PR TITLE
VB-927: Fix issue where closed visits don't display on visits page if…

### PR DESCRIPTION
**Pull request by psoleckimoj**
… there are no open visits

### CODE REVIEW

File: server/services/visitSessionsService.ts [hunk 1]
```diff
@@ -389,7 +329,7 @@ export default class VisitSessionsService {
       startTimestamp: visit.startTimestamp,
       visitDate: prisonerDateTimePretty(visit.startTimestamp),
       visitTime,
-      visitRestriction: visit.visitRestriction,
+      visitRestriction: visit.visitRestriction === 'OPEN' ? 'OPEN' : 'CLOSED',
```
Comment (psoleckimoj) [inline server/services/visitSessionsService.ts:332]: This is because imported visits are sometimes UNKNOWN, not sure what we should default to tbh.

Comment (psoleckimoj) [reply server/services/visitSessionsService.ts:332]: Have asked the question, this may change to OPEN

File: server/utils/visitsUtils.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+): {
+  openSlots: VisitsPageSlot[]
+  closedSlots: VisitsPageSlot[]
+  firstSlotTime: string
+} => {
+  const openSlots: VisitsPageSlot[] = []
+  const closedSlots: VisitsPageSlot[] = []
+
+  visits.forEach((visit: ExtendedVisitInformation) => {
+    if (visit.visitRestriction === 'OPEN') {
```
Comment (psoleckimoj) [inline server/utils/visitsUtils.ts:15]: Again this may change to OPEN or UNKNOWN

File: server/services/visitSessionsService.ts (hunk 1 above)
Comment (tpmcgowan) [reply server/services/visitSessionsService.ts:332]: Similar to what's on https://dsdmoj.atlassian.net/browse/VB-995 - I raised the question of what to do with 'Unknown' visits. Feels like that issue should be linked to VB-927?

Comment (psoleckimoj) [reply server/services/visitSessionsService.ts:332]: @tpmcgowan yep, Phoebe is going to link them as it all relates to how to display them.

**Review by tpmcgowan — APPROVED** [review]
LGTM

**Pull request merged**